mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -929,6 +929,33 @@ public File downloadLatestModelVersion(String modelName, String stage) {
       return downloadModelVersion(modelName, details.getVersion());
   }
 
+  /**
+   * Return model versions.
+   *
+   * @return A page of model versions.
+   */
+  public ModelVersionsPage searchModelVersions() {
+    return searchModelVersions("");
+  }
+
+  /**
+   * Return model versions that satisfy the search query.
+   *
+   * @param searchFilter SQL compatible search query string.
+   *                     Examples:
+   *                         - "name = 'model_name'"
+   *                         - "run_id = '...'"
+   *                     If null, the result will be equivalent to having an empty search filter.
+   *
+   * @return A page of model versions that satisfy the search filter.
+   */
+  public ModelVersionsPage searchModelVersions(String searchFilter) {
```

Comment:
could we update this with the recently added `order_by`, `max_results`, and `page_token` arguments that the Python API now supports?

Reply:
Updates can be seen in this PR: https://github.com/mlflow/mlflow/pull/7623

Reply:
Oh I missed it! Of course, let me update this.